roles/permissions.py

```diff
@@ -0,0 +1,31 @@
+"""
+Specific permissions for the micromasters app
+"""
+
+from rolepermissions.permissions import register_object_checker
+from rolepermissions.verifications import has_permission
+
+from roles.models import Role
+from roles.roles import Permissions
+
+
+@register_object_checker()
+def can_advance_search(role, user, program):
+    """
+    Specific permission checker on a program
+    """
+    return (
+        has_permission(user, Permissions.CAN_ADVANCE_SEARCH) and Role.objects.filter(
+            user=user, role=role.ROLE_ID, program=program).exists()
+    )
+
+
+@register_object_checker()
+def can_message_learners(role, user, program):
+    """
+    Specific permission checker on a program
```

Comment:
can you make these docstrings more specific? eg: "Determines whether a user can perform an advanced search" and "Determines whether a user can send a message to learners".